x/gov/client/cli/prompt.go

```diff
@@ -63,9 +63,11 @@ func Prompt[T any](data T, namePrefix string) (T, error) {
 	}
 
 	for i := 0; i < v.NumField(); i++ {
-		if v.Field(i).Kind() == reflect.Struct || v.Field(i).Kind() == reflect.Slice {
-			// if the field is a struct skip
-			// in a future we can add a recursive call to Prompt
+		// if the field is a struct skip or not slice of string or int then skip
+		// TODO in the future we can add a recursive call to Prompt
+		if v.Field(i).Kind() == reflect.Struct ||
+			(v.Field(i).Kind() == reflect.Slice &&
+				(v.Field(i).Type().Elem().Kind() != reflect.String && v.Field(i).Type().Elem().Kind() != reflect.Int)) {
```

Comment:
silly nit: we could simplify towards readability by separating this into a couple of if statements

Reply:
sure, using a switch instead 👍🏾